training_script.py

```diff
@@ -0,0 +1 @@
+#TODO: MAKE TRAINING SCRIPT
```

Comment:
Please rename to sth simpler like "train.py". It is obvious that this is a script.